.github/workflows/container-image-build.yml

```diff
@@ -135,6 +135,10 @@ jobs:
           retention-days: 7
         if: github.event.input.seed
 
+      - name: Prune local Kolla container images
+        run: |
+          docker image prune --filter="label=kolla_version"
```

Comment:
Use the force, Mark.

```
-f, --force           Do not prompt for confirmation
```

Also, you are using `sudo` for the other docker commands, don't you need it here too?

Reply:
The force is strong in this one